documentation/modules/operators/proc-configuring-kafka-topic.adoc

```diff
@@ -3,24 +3,25 @@
 // assembly-using-the-topic-operator.adoc
 
 [id='proc-configuring-kafka-topic-{context}']
-= Configuring a Kafka topic
+= Configuring Kafka topics
 
-Use the properties of the `KafkaTopic` resource to configure a Kafka topic.
+[role="_abstract"]
+Use the properties of the `KafkaTopic` resource to configure Kafka topics.
 
 You can use `kubectl apply` to create or modify topics, and `kubectl delete` to delete existing topics.
 
 For example:
 
-* `kubectl apply -f <topic-config-file>`
-* `kubectl delete KafkaTopic <topic-name>`
+* `kubectl apply -f _<topic_config_file>_`
+* `kubectl delete KafkaTopic _<topic_name>_`
 
 This procedure shows how to create a topic with 10 partitions and 2 replicas.
 
 .Before you start
 
 It is important that you consider the following before making your changes:
 
-* Kafka does _not_ support making the following changes through the `KafkaTopic` resource:
+* Kafka *does not* support making the following changes through the `KafkaTopic` resource:
 ** Changing topic names using `spec.topicName`
 ** Decreasing partition size using `spec.partitions`
 * You cannot use `spec.replicas` to change the number of replicas that were initially specified.
```

Comment:
I think tis is a bit weird. It says that Kafka does not support these things. But Kafka does not know about KafkaTopics, so it makes little sense.

`Changing topic names` and `changing the number of replicas` are purely Strimzi things. `Decreasing partition size` is a Kafka thing. So maybe they should be presented separately.

Reply:
Changed to:

> * Kafka does not support decreasing the number of partitions.
> * Increasing `spec.partitions` for topics with keys will change how records are partitioned, which can be particularly problematic when the topic uses _semantic partitioning_.
> * Strimzi does not support making the following changes through the `KafkaTopic` resource:
> ** Using `spec.replicas` to change the number of replicas that were initially specified
> ** Changing topic names using `spec.topicName`